BLSQ/iaso
# IA-2509: Upgrade material ui

Remove reference to material ui 4 and implement MUI 5

Related JIRA tickets : IA-2509

## Self proofreading checklist


## Changes

- upgrade `bluesquare-components` to use MUI5
- rewrite `SnackBarContainer` to a hook in TSX
- replace all occurrences of @materila-ui
- small css fixes

## How to test

[Cypress run](https://github.com/BLSQ/iaso/actions/runs/6744721996/job/18335703039)
Delete your node modules or webpack container container / image
Run `npm ci` or `docker-compose build`
Make sure is working, try to go a bit everywhere


## Notes

⚠️ This is using a branch of bluesquare-components
